### PR TITLE
[new chart] Citus Postgresql

### PR DESCRIPTION
**Pull request by bequiet88**
#### What this PR does / why we need it:
This PR adds a Helm chart to bootstrap a [Citus Postgresql Cluster](https://www.citusdata.com/). Citus is a Postgres extension (not a fork) that intelligently distributes data and queries across nodes so that the database can scale and execute queries fast. The [Citus Membership Manager](https://github.com/bakdata/citus-k8s-membership-manager) adds automated scaling as well as provisioning functionality.

#### Special notes for your reviewer:
The commiters are not affiliated in any way with Citus. We just needed a convenient way to run Citus on Kubernetes and we'd like to contribute our Helm chart back to the community.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by bequiet88** [comment]
keeping it open

**Comment by bequiet88** [comment]
keeping it open

File: incubator/citus-postgresql/README.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+## Prerequisites
+
+* Kubernetes 1.10.0+
+* Helm 2.10.0+
+
+## Docker Image Source
+
+* [Citus](https://github.com/citusdata/docker)
+* [Citus Membership Mamanger](https://github.com/bakdata/citus-k8s-membership-manager)
```
Comment (neil-pozetroninc) [inline incubator/citus-postgresql/README.md]: You have a small typo in the link. 

`Mamanger` -> `Manager`

Comment (bequiet88) [reply incubator/citus-postgresql/README.md]: Thanks for pointing out, Neil.

Comment (bequiet88) [reply incubator/citus-postgresql/README.md]: fixed

**Comment by jonathanmeier5** [comment]
Any news here? Very excited to use this.

**Comment by maorfr** [comment]
hey @bequiet88 

Have you considered hosting this chart and adding it to helm hub?
https://github.com/helm/hub/blob/master/Repositories.md

**Comment by bequiet88** [comment]
keeping it open

**Comment by Aisuko** [comment]
@bequiet88 This was absolute good work. Would you like to join the Kubernetes/charts channels? You would get more feedback on this PR.

**Comment by bequiet88** [comment]
@Aisuko thanks :-) what kind of channel is Kubernetes/charts? Slack? Where to join?

**Comment by bequiet88** [comment]
@maorfr that might be an option, once we decide to open-source all our other internal helm charts. I'm keeping it in my mind.

**Comment by Aisuko** [comment]
> @Aisuko thanks :-) what kind of channel is Kubernetes/charts? Slack? Where to join?

https://github.com/helm/charts#where-to-find-us

**Comment by maorfr** [comment]
please take a look at https://github.com/helm/helm/pull/6095#issue-301608768

we strongly encourage people to self host chart repos:
https://github.com/helm/hub/blob/master/Repositories.md

you can even do that using GitHub pages:
https://github.com/helm/hub/blob/master/docs/github.md#github-pages-as-a-repository

**Comment by lfdominguez** [comment]
Hey, anyone can approve this???

**Comment by Aisuko** [comment]
@lfdominguez Hello sir, join the charts channel to increase interaction could be good. The channel address on the README.md.

**Comment by lfdominguez** [comment]
Hey, anyone can approve this???

> @lfdominguez Hello sir, join the charts channel to increase interaction could be good. The channel address on the README.md.

ggg one big problem... EEUU block Slack for Cuba... so.... i can't access.

**Comment by Aisuko** [comment]
> Hey, anyone can approve this???
> 
> > @lfdominguez Hello sir, join the charts channel to increase interaction could be good. The channel address on the README.md.
> 
> ggg one big problem... EEUU block Slack for Cuba... so.... i can't access.

Yes, Slack was blocked by China xxx sometimes.  @maorfr  I'm sorry to bother you sir, how can we do any other things on this PR?

**Comment by zanhsieh** [comment]
@davidkarlsen 
Would you mind to help this chart please?

**Comment by LeeWong** [comment]
any update on this?

**Comment by maorfr** [comment]
https://github.com/helm/charts/pull/10618#issuecomment-520708117

**Comment by bequiet88** [comment]
> 
> 
> [#10618 (comment)](https://github.com/helm/charts/pull/10618#issuecomment-520708117)

Okay, I'll have a look at this opportunity.

**Comment by scottrigby** [comment]
Hi 👋 new charts are no longer accepted to `stable` or `incubator`. See [Status of the Project](https://github.com/helm/charts#status-of-the-project) and [Deprecation Timeline](https://github.com/helm/charts#deprecation-timeline) for more info about that.

Good news is you can host a Helm chart yourself on GitHub, use the same tools as this repo does for automatic Chart [testing](https://github.com/helm/chart-testing) and [releasing](https://github.com/helm/chart-releaser), and list your Helm repo on https://helm.hub.sh. Follow this issue for info about existing guidelines and upcoming improvements: https://github.com/helm/charts/issues/7800

**Comment by scottrigby** [comment]
@bequiet88 would love to see this on the Hub. Ping us on Slack if you need any help with the above.

**Pull request closed without merging**